src/flux/nylas-api.coffee

```diff
@@ -151,6 +151,9 @@ class NylasAPI
     else if env in ['local']
       @AppID = 'n/a'
       @APIRoot = 'http://localhost:5555'
+    else if env in ['custom']
+      @AppID = NylasEnv.config.get('env.AppID') or 'n/a'
```

Comment:
Hey @mbilker — I'm not 100% sure that the config schema will allow sub-keys on `env` because it's expected to be a string. Was this working? Might need to be envAppId or another separate key.